### PR TITLE
temporarily disable ncclBcastOp test, it fails randomly

### PR DESCRIPTION
**Pull request by helinwang**

### CODE REVIEW

File: paddle/fluid/operators/nccl_op_test.cu.cc [hunk 1]
```diff
@@ -236,48 +236,49 @@ TEST_F(NCCLTester, ncclReduceOp) {
 }
 
 // ncclBcastOp with desc
```
Comment (wangkuiyi) [inline paddle/fluid/operators/nccl_op_test.cu.cc]: Should we have an issue to remind us about this temporary disabling?

And how about adding a comment here mentioning the issue?

File: paddle/fluid/operators/nccl_op_test.cu.cc [hunk 2]
```diff
@@ -236,48 +236,49 @@ TEST_F(NCCLTester, ncclReduceOp) {
...
-  paddle::memory::Copy(
-      cpu_place, ct, p::CUDAPlace(gpu_list_[idx]), rt,
-      recv_tensor.numel() * sizeof(float),
-      static_cast<p::CUDADeviceContext *>(dev_ctxs_[idx])->stream());
-
-  for (int64_t j = 0; j < f::product(kDims); ++j) {
-    ASSERT_NEAR(ct[j], result, 1e-5);
-  }
-}
+// TODO(helin): enable the test for ncclBcastOp
```
Comment (wangkuiyi) [inline paddle/fluid/operators/nccl_op_test.cu.cc]: How about `/* */` instead of `//`?

Comment (helinwang) [reply paddle/fluid/operators/nccl_op_test.cu.cc]: Good idea!

File: paddle/fluid/operators/nccl_op_test.cu.cc (hunk 1 above)
Comment (helinwang) [reply paddle/fluid/operators/nccl_op_test.cu.cc]: Yes, it's here: https://github.com/PaddlePaddle/Paddle/issues/9540 , will add it to comment.

Comment (helinwang) [reply paddle/fluid/operators/nccl_op_test.cu.cc]: Done.

File: paddle/fluid/operators/nccl_op_test.cu.cc (hunk 2 above)
Comment (helinwang) [reply paddle/fluid/operators/nccl_op_test.cu.cc]: Done.

**Comment by helinwang** [comment]
disabled ncclAllReduceOp according to @luotao1 's comment as well: https://github.com/PaddlePaddle/Paddle/issues/9540#issuecomment-377663378

**Pull request merged**